addons/l10n_in_ewaybill_stock/security/ir.model.access.csv

```diff
@@ -1,3 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-l10n_in_ewaybill_stock.access_l10n_in_ewaybill,access_l10n_in_ewaybill,l10n_in_ewaybill_stock.model_l10n_in_ewaybill,stock.group_stock_manager,1,1,1,1
-l10n_in_ewaybill_stock.access_l10n_in_ewaybill_cancel,access_l10n_in_ewaybill_cancel,l10n_in_ewaybill_stock.model_l10n_in_ewaybill_cancel,stock.group_stock_manager,1,1,1,1
+l10n_in_ewaybill.access_l10n_in_ewaybill,access_l10n_in_ewaybill_stock,l10n_in_ewaybill.model_l10n_in_ewaybill,stock.group_stock_manager,1,1,1,1
```

Comment:
Not sure that it's a good idea to change the original rules.

Reply:
Indeed. My bad. It should be `l10n_in_ewaybill_stock.access_l10n_in_ewaybil`